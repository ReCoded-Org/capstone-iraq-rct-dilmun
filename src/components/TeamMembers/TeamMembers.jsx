import React from 'react'
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
import { useTranslation } from 'react-i18next'
import havalImg from '../../assets/haval.jfif'

export default function TeamMembers() {
  const { t } = useTranslation()

  return (
    <div className="bg-white p-8">
      <div className="text-4xl mb-10 text-center my-5 text-darkBlue font-bold  ">
        {' '}
        {t('about.meet')}
      </div>

      <div className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-2 xl:grid-cols-4 gap-4 mb-14 mx-4">
        <div className="flex flex-col text-center items-center justify-center bg-white p-4 shadow rounded-lg">
          <div className="inline-flex shadow-lg border rounded-full overflow-hidden h-40 w-40">
            <img src={havalImg} alt="" />
          </div>

          <h2 className="mt-4 font-bold text-xl">{t('about.haval')}</h2>

          <ul className="flex flex-row mt-4 space-x-2">
            <li>
              <a
                href="https://github.com/haval95"
                className="flex hover:bg-blue items-center justify-center h-8 w-8 border rounded-full "
              >
                <FontAwesomeIcon icon={['fab', 'github']} />
              </a>
            </li>
          </ul>
        </div>

        <div className="flex flex-col items-center justify-center bg-white p-4 shadow rounded-lg">
          <div className="inline-flex shadow-lg border rounded-full overflow-hidden h-40 w-40">
            <FontAwesomeIcon
              icon="user"
              className="h-full w-full fa-6x mx-auto justify-center items-center"
            />
          </div>

          <h2 className="mt-4 font-bold text-xl">{t('about.ismail')}</h2>
          <ul className="flex flex-row mt-4 space-x-2">
            <li>
              <a
                href="https://github.com/ismailkhalil20"
                className="flex hover:bg-blue  items-center justify-center h-8 w-8 border rounded-full "
              >
                <FontAwesomeIcon icon={['fab', 'github']} />
              </a>
            </li>
          </ul>
        </div>

        <div className="flex flex-col items-center justify-center bg-white p-4 shadow rounded-lg">
          <div className="inline-flex shadow-lg border rounded-full overflow-hidden h-40 w-40">
            <FontAwesomeIcon
              icon="user"
              className="h-full w-full fa-6x mx-auto justify-center items-center"
            />
          </div>

          <h2 className="mt-4 text-center font-bold text-xl">
            {t('about.sima')}
          </h2>
          <ul className="flex flex-row mt-4 space-x-2">
            <li>
              <a
                href="https://github.com/Sima-J"
                className="flex hover:bg-blue  items-center justify-center h-8 w-8 border rounded-full "
              >
                <FontAwesomeIcon icon={['fab', 'github']} />
              </a>
            </li>
          </ul>
        </div>

        <div className="flex flex-col items-center justify-center bg-white p-4 shadow rounded-lg">
          <div className="inline-flex shadow-lg border rounded-full overflow-hidden h-40 w-40">
            <FontAwesomeIcon
              icon="user"
              className="h-full w-full fa-6x mx-auto justify-center items-center"
            />
          </div>

          <h2 className="mt-4 font-bold text-xl">{t('about.kodo')} </h2>
          <ul className="flex flex-row mt-4 space-x-2">
            <li>
              <a
                href="https://github.com/kodo-yousif"
                className="flex hover:bg-blue items-center justify-center h-8 w-8 border rounded-full "
              >
                <FontAwesomeIcon icon={['fab', 'github']} />
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  )
}
