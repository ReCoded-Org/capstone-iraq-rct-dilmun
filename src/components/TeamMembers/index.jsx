export { default } from './TeamMembers'
export { default as AboutUs } from './AboutUs'
