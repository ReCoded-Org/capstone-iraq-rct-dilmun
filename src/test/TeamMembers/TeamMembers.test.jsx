import React from 'react'
import renderer from 'react-test-renderer'
import { BrowserRouter as Router } from 'react-router-dom'
import TeamMembers from '../../components/TeamMembers';

it('Test: TeamMembers  passed', () => {
  const tree = renderer
    .create(
      <Router>
        <TeamMembers />
      </Router>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
