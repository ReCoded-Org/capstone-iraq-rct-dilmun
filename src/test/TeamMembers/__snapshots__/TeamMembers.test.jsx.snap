// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test: TeamMembers  passed 1`] = `
<div
  className="bg-white p-8"
>
  <div
    className="text-4xl mb-10 text-center my-5 text-darkBlue font-bold  "
  >
      
    about.meet
  </div>
  <div
    className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-2 xl:grid-cols-4 gap-4 mb-14 mx-4"
  >
    <div
      className="flex flex-col text-center items-center justify-center bg-white p-4 shadow rounded-lg"
    >
      <div
        className="inline-flex shadow-lg border rounded-full overflow-hidden h-40 w-40"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-user fa-w-14 h-full w-full fa-6x mx-auto justify-center items-center"
          data-icon="user"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 448 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </div>
      <h2
        className="mt-4 font-bold text-xl"
      >
        about.haval
      </h2>
      <ul
        className="flex flex-row mt-4 space-x-2"
      >
        <li>
          <a
            className="flex hover:bg-blue items-center justify-center h-8 w-8 border rounded-full "
            href="https://github.com/haval95"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-github fa-w-16 "
              data-icon="github"
              data-prefix="fab"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 496 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </a>
        </li>
      </ul>
    </div>
    <div
      className="flex flex-col items-center justify-center bg-white p-4 shadow rounded-lg"
    >
      <div
        className="inline-flex shadow-lg border rounded-full overflow-hidden h-40 w-40"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-user fa-w-14 h-full w-full fa-6x mx-auto justify-center items-center"
          data-icon="user"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 448 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </div>
      <h2
        className="mt-4 font-bold text-xl"
      >
        about.ismail
      </h2>
      <ul
        className="flex flex-row mt-4 space-x-2"
      >
        <li>
          <a
            className="flex hover:bg-blue  items-center justify-center h-8 w-8 border rounded-full "
            href="https://github.com/ismailkhalil20"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-github fa-w-16 "
              data-icon="github"
              data-prefix="fab"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 496 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </a>
        </li>
      </ul>
    </div>
    <div
      className="flex flex-col items-center justify-center bg-white p-4 shadow rounded-lg"
    >
      <div
        className="inline-flex shadow-lg border rounded-full overflow-hidden h-40 w-40"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-user fa-w-14 h-full w-full fa-6x mx-auto justify-center items-center"
          data-icon="user"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 448 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </div>
      <h2
        className="mt-4 text-center font-bold text-xl"
      >
        about.sima
      </h2>
      <ul
        className="flex flex-row mt-4 space-x-2"
      >
        <li>
          <a
            className="flex hover:bg-blue  items-center justify-center h-8 w-8 border rounded-full "
            href="https://github.com/Sima-J"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-github fa-w-16 "
              data-icon="github"
              data-prefix="fab"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 496 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </a>
        </li>
      </ul>
    </div>
    <div
      className="flex flex-col items-center justify-center bg-white p-4 shadow rounded-lg"
    >
      <div
        className="inline-flex shadow-lg border rounded-full overflow-hidden h-40 w-40"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-user fa-w-14 h-full w-full fa-6x mx-auto justify-center items-center"
          data-icon="user"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 448 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </div>
      <h2
        className="mt-4 font-bold text-xl"
      >
        about.kodo
         
      </h2>
      <ul
        className="flex flex-row mt-4 space-x-2"
      >
        <li>
          <a
            className="flex hover:bg-blue items-center justify-center h-8 w-8 border rounded-full "
            href="https://github.com/kodo-yousif"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-github fa-w-16 "
              data-icon="github"
              data-prefix="fab"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 496 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </a>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
